chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -715,14 +716,14 @@ where
         // take the biggest period possible.
         let period = BlockHashCount::get()
             .checked_next_power_of_two()
-            .map(|c| c / 2)
+            .map(|c| c.saturating_div(2))
```

Comment:
This function is useless because it still panics when dividing by 0, I propose we ban it.

Can you try to do that? Using https://rust-lang.github.io/rust-clippy/master/#disallowed_methods

And here use either checked_div or nothing, I think nothing looks better

```suggestion
            .map(|c| c / 2)
```
```suggestion
            .map(|c| c.checked_div(2).expect("2 != 0; qed"))
```

Reply:
agreed